# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iptables/ranlib.patch.cross
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- iptables-1.3.5/extensions/Makefile.orig	2006-03-31 15:40:33.000000000 +0200
+++ iptables-1.3.5/extensions/Makefile	2006-03-31 15:40:51.000000000 +0200
@@ -67,10 +67,10 @@
 
 ifdef NO_SHARED_LIBS
 extensions/libext.a: $(EXT_OBJS)
-	rm -f $@; ar crv $@ $(EXT_OBJS)
+	rm -f $@; $(AR) crvs $@ $(EXT_OBJS)
 
 extensions/libext6.a: $(EXT6_OBJS)
-	rm -f $@; ar crv $@ $(EXT6_OBJS)
+	rm -f $@; $(AR) crvs $@ $(EXT6_OBJS)
 
 extensions/initext.o: extensions/initext.c
 extensions/initext6.o: extensions/initext6.c
